scripts/cli.js

```diff
@@ -7,38 +7,51 @@ const Kredits = require('../lib/kredits');
 module.exports = function(callback) {
   const Registry = artifacts.require('./Registry.sol');
   Registry.deployed().then(async (registry) => {
-    let contractName = await promptly.prompt('Contract Name: ');
-    let method = await promptly.prompt('Function: ');
-    let argumentInput = await promptly.prompt('Arguments (comma separated): ', { default: '' });
-    let args = [];
-    if (argumentInput !== '') {
-      args = argumentInput.split(',').map(a => a.trim());
-    }
-
     const networkId = parseInt(web3.version.network);
     const provider = new ethers.providers.Web3Provider(
       web3.currentProvider, { chainId: networkId }
     );
     const kredits = await Kredits.setup(provider, provider.getSigner());
 
-    const contract = kredits[contractName].contract;
-    console.log(`Using ${contractName} at ${contract.address}`);
-    console.log(`Calling ${method} with ${JSON.stringify(args)}`);
+    let contractName = await promptly.prompt('Contract Name: ');
+    const contractWrapper = kredits[contractName];
```

Comment:
So good that we went with upper case contract names and the properties instead of the functions!
But it will breaks on `Contributor`, wouldn’t it? I think we should rename that contract next and align the module code to it. Will make the future much brighter!

Reply:
yes, agree on renaming it! and maybe adding an alias for Contributor <=> Contributors

Reply:
I opened #44 for the alias